core/src/main/java/org/elasticsearch/action/admin/cluster/node/info/PluginsInfo.java

```diff
@@ -45,7 +47,17 @@ public PluginsInfo(int size) {
         infos = new ArrayList<>(size);
     }
 
+    /**
+     * @return an ordered list based on plugins name
+     */
     public List<PluginInfo> getInfos() {
+        Collections.sort(infos, new Comparator<PluginInfo>() {
+            @Override
+            public int compare(final PluginInfo o1, final PluginInfo o2) {
+                return o1.compare(o2);
```

Comment:
lol. That's what I did at first but I was wondering if it's a better practice to have that in the Class itself... :p